x-pack/plugins/uptime/server/lib/requests/get_network_events.test.ts

```diff
@@ -5,9 +5,19 @@
  */
 
 import { getUptimeESMockClient } from './helper';
-import { getNetworkEvents } from './get_network_events';
+import { getNetworkEvents, secondsToMillis } from './get_network_events';
 
 describe('getNetworkEvents', () => {
+  describe('secondsToMillis conversion', () => {
+    it('returns -1 for -1 value', () => {
+      expect(secondsToMillis(-1)).toBe(-1);
+    });
+
+    it('returns a value of seconds as milliseconds', () => {
+      expect(secondsToMillis(10)).toBe(10_000);
```

Comment:
I have never in my life seen this underscore thing. Is that just syntactic sugar for formatting numbers?

Reply:
'tis. Isn't it freaky? Someone on Twitter told me about it the other day.

Reply:
PS @dominiqueclarke you only commented, not approved. Please ✅ if still LGTY.